source/manage/client-annotations.rst

```diff
@@ -42,7 +42,50 @@ They can be set globally or for specific nodes or services.
 ``--service-name``
     The service name being annotated.
 
-.. _application-event-marking:
+***************
+Combining flags
+***************
+
+Flags may be combined as shown in the following examples.
+
+``--node``
+    current node
+
+``--node-name``
+    node with name
+
+``--node --node-name``
+    node with name
+
+``--node --service-name``
+    current node and service with name
+
+``--node --node-name --service-name``
```

Comment:
maybe we can wrap this and other cases to say "if  node  or service name is specified  - they will be used  instead of other parameters "